content/python/concepts/requests-module/terms/get/get.md

```diff
@@ -14,23 +14,23 @@ CatalogContent:
   - 'paths/analyze-data-with-python'
 ---
 
-Method for making a "get" request to a web server; it returns a response object.
+The **`.get()`** method sends a GET request to a web server; it returns a response object.
 
 ## Syntax
 
-```python
+```py
 import requests
 
 requests.get("url", **kwargs)
 ```
 
-`**kwargs` are any number of dictionary items (named arguments) that are passed in as parameters.
+`**kwargs` are any number of dictionary items (named arguments) that are passed in as parameters. Many different named parameters can be passed in to a GET request. For example, they can be used to include cookies in the request, set proxies, set user-agents, or set a page timeout.
 
-## Example 1
+## Example
 
 The `.get()` method sends a request for data to a web server. The response object it returns contains various types of data such as the webpage text, status code, and the reason for that response.
 
-```python
+```py
 import requests
 
 response = requests.get("https://codecademy.com")
```

Comment:
Within lines # 38 through 50, I would consolidate the `status_code` and `reason` into a single `print(f"")` statement, like so:

```py
print(f"{response.status_code}: {response.reason}")
```

Then, I suggest revising/reallocating the `# Output: ` comments like this:

```shell
200: OK
404: Not Found
```